fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -742,3 +761,35 @@ private[this] final class Trie[T](strings: Seq[(String, T)]){
   }
 }
 
+
+
+trait TrueColor extends Category {
+  /**
+    * 38 : front color
+    * 48 : background color
+    */
+  val zindex : Int
```

Comment:
I had take my inspiration in css rule, but according to wikipedia this attribute must be call Selected Graphic Rendition
do you prefer `str` or `selectedGraphicRendition` ?